src/test/java/org/dependencytrack/resources/v1/ProjectResourceTest.java

```diff
@@ -1046,6 +1097,60 @@ public void getProjectChildrenConciseFilterByTagTest() {
                 """);
     }
 
+    @Test
+    public void getProjectChildrenConciseFilterByTeamTest() {
+        final var parentProject = new Project();
+        parentProject.setName("acme-app");
+        qm.persist(parentProject);
+
+        final var childProjectA = new Project();
+        childProjectA.setParent(parentProject);
+        childProjectA.setName("acme-child-app-a");
+        qm.persist(childProjectA);
+
+        final var childProjectB = new Project();
+        childProjectB.setParent(parentProject);
+        childProjectB.setName("acme-child-app-b");
+        qm.persist(childProjectB);
+
+        childProjectB.addAccessTeam(team);
+
+        // Should not return results for partial matches.
+        Response response = jersey.target(V1_PROJECT + "/concise/" + parentProject.getUuid() + "/children")
+                .queryParam("team", "f")
+                .request()
+                .header(X_API_KEY, apiKey)
+                .get();
+        assertThat(response.getStatus()).isEqualTo(200);
+        assertThat(response.getHeaderString(TOTAL_COUNT_HEADER)).isEqualTo("0");
+        assertThat(getPlainTextBody(response)).isEqualTo("[]");
+
+        // Should return results for exact matches.
+        response = jersey.target(V1_PROJECT + "/concise/" + parentProject.getUuid() + "/children")
+                .queryParam("team", team.getName())
+                .request()
+                .header(X_API_KEY, apiKey)
+                .get();
+        assertThat(response.getStatus()).isEqualTo(200);
+        assertThat(response.getHeaderString(TOTAL_COUNT_HEADER)).isEqualTo("1");
+        assertThatJson(getPlainTextBody(response)).isEqualTo("""
+                [
+                  {
+                    "uuid": "${json-unit.any-string}",
+                    "name": "acme-child-app-b",
+                    "active": true,
+                    "teams": [
+                      {
+                        "name": "foo"
```

Comment:
The format string `%s` is missing here:

```suggestion
                        "name": "%s"
```